prog/vnet/update_firewall_rules.rb

```diff
@@ -23,6 +23,12 @@ def before_run
       <<~LOAD_BALANCER_ALLOW_RULE
 #{allow_ipv4_lb_neigh_incoming}
 #{allow_ipv6_lb_neigh_incoming}
+
+# The traffic that is routed to the local VM from the load balancer
+# is marked with 0x00B1C100D. We need to allow this traffic to
+# the local VM.
+meta mark 0x00B1C100D ip saddr . tcp dport @allowed_ipv4_lb_dest_set ct state established,related,new counter accept
```

Comment:
oobiclood shows up again! I was wondering if we were going to get into packet marking.